frb/figures/finder.py

```diff
@@ -109,8 +114,7 @@ def generate(image, wcs, title, log_stretch=False,
     cimg = ax.imshow(image, cmap='Greys', norm=norm)
 
     # Flip so RA increases to the left
-    ax.invert_xaxis()
-
+    #ax.invert_xaxis()
```

Comment:
Please make this option, e.g. `invert`.  I have a feeling we will want to 
use it when we make one in the Southern hemisphere.